### DISCUSSION TITLE
the text is shifting/moving down when generating PDF

### DISCUSSION THREAD

**Discussion by phojie** [tailwindlabs/tailwindcss#7870] [Help]
**What version of Tailwind CSS are you using?**
v3

**What build tool (or framework if it abstracts the build tool) are you using?**
Vue 2 / Nuxt 2 
JsPdf

**Issue**
1. The displayed text is moving down after generating the component to PDF
2. This only happens after I added tailwindcss. I tried some several css frameworks, It all work so well. 

Without Tailwind 
<img width="447" alt="Screen Shot 2022-03-17 at 2 09 44 AM" src="https://user-images.githubusercontent.com/33350692/158658387-fafd5133-237e-4407-b881-0d055b226876.png">
-ba8d-39785638e2e9.png">

With Tailwind
<img width="382" alt="Screen Shot 2022-03-17 at 2 09 08 AM" src="https://user-images.githubusercontent.com/33350692/158658278-bdba108c-adb5-4ca4-9550-2c91b742bd36.png">

**Comment by Bobybyk** [comment]
Same problem here, i don't understand why...

**Reply by L9m** [reply]
disabled taliwind preflight work for me
```
corePlugins: {
    boxSizing: 'border-box',
    preflight: false,
  },
```

**Comment by smithliam** [comment]
Provide something to test in the playground https://play.tailwindcss.com/ after replicating the same problem in the playground, simply copy the link via the Share button and paste it here.

**Comment by JakeBelanger** [comment]
Did you ever figure it out?

**Comment by alts9** [comment]
Apparently, this is caused by Tailwind Preflight (similar to reset CSS). I managed to undo the effect by inserting this into my CSS:
`img,
svg,
video,
canvas,
audio,
iframe,
embed,
object {
  display: initial;
  vertical-align: initial;
}` 

In [preflight stylesheet](https://unpkg.com/tailwindcss@%5E3/src/css/preflight.css), the value changed to '`display: block; 
  vertical-align: middle;` so override this setting might affect Tailwind's behavior

[Read Tailwind docs about Preflight for more info](https://tailwindcss.com/docs/preflight)

**Reply by jp1304-brikl** [reply]
This works like a charm! Thanks a lot!
